mail/mpack/Portfile

```
@@ -2,7 +2,7 @@ PortSystem		1.0
 
 name            	mpack
 version         	1.6
-revision		1
+revision		2
```

As explained in the guide, don't increase the revision just to fix a build failure since nothing has changed for anyone (i.e. those with older less-picky compilers) who was already able to install the port (unless using autoreconf causes more changes for this software than I'm used to).
```suggestion
revision		1
```